src/utils.js

```diff
@@ -138,3 +138,15 @@ export function numberSortAsc(a, b) {
 export function stripHTML(html) {
     return html.replace(/<[^>]*>/g, '');
 };
+
+export function format(str, args) {
+    if (!str) return str;
+
+    return str.replace(/{(\d+)}/g, function (match, number) {
+        return typeof args[number] !== 'undefined' ? args[number] : match;
+    });
+};
+
+export function pluralize(word, count, suffix) {
+    return `${word}${count > 1 ? suffix : ''}`;
+};
```

Comment:
`pluralize` doesn't always mean adding a suffix. For e.g., `la pera` is singular and `las peras` is plural.

better design would be
```
pluralize('la pera', 'las peras', 2)
```

Reply:
@netchampfaris @barredterra added the pluralization in the translations itself, can you have a look at it